src/htr2hpc/train/data.py

```diff
@@ -147,7 +170,9 @@ def get_model(api, model_id, training_type, output_dir):
     return api.download_file(model_info.file, output_dir)
 
 
-def get_training_data(api, output_dir, document_id, part_ids=None):
+def get_training_data(
+    api, output_dir, document_id, part_ids=None, transcription_id=None
+):
     # if part ids are not specified, get all parts
     if part_ids is None:
         doc_parts = api.document_parts_list(document_id)
```

Comment:
This seems to currently only be getting all the results from the first page of API results, and not accounting for further pages. I tried test running transcription training on the full document in [doc 27](https://test-htr.lib.princeton.edu/document/27/images/) -- by default, this should have downloaded all 60 parts of the document. But it only downloaded the first 10 images (the first page of API results).